### PR TITLE
[Doc]java importer doc

### PR DESCRIPTION
**Pull request by jude-zhu**
java importer doc

### CODE REVIEW

File: src/tools/importer/README.md [hunk 1]
```diff
@@ -1,33 +1,161 @@
...
+
+#### For Edge Types
+
+For edge type, the file structure is as following:
+
+```
+sourceID, destinationID, [ranking], prop_name_1, prop_name_2, ...
+```
+
+> The "ranking" column is optional. It works with "--ranking" optional, which will be described below.
```
Comment (Amber1990Zhang) [inline src/tools/importer/README.md]: optional >- option
I think we need a noun rather than adj. here.

File: src/tools/importer/README.md [hunk 2]
```diff
@@ -1,33 +1,161 @@
...
+|--name           | -n            | specify the space name.| myspace_test2 |
+|--schema         | -m            | specify the tag or edgetype name.| student |
+|--column         | -c            | properties of tag or edge to be inserted, separated by a comma | name,age |
+|--ranking        | -k            | whether the edge has ranking value. Default is false| true/false|
+|--file           | -f            | data file| ./tmp/data.txt |
+|--errorPath    | -d            | error log file | ./tmp/error.log |
+|--help           | -h            | list help|
+|--timeout        | -o            | specify connection timeout, in millisecond| 3000 |
+|--pswd           | -p            | graphd service password||
+|--connectionRetry       | -r            | connection retry times|
```
Comment (Amber1990Zhang) [inline src/tools/importer/README.md]: example?

File: src/tools/importer/README.md [hunk 3]
```diff
@@ -1,33 +1,161 @@
...
+|--executionRetry       | -e           | thrift execution retry times|
+|--stat           | -s            | print statistics info| true/false |
+|--type           | -t            | indicate to insert vertex properties or edge properties| vertext/edge|
+|--user           | -u            | graphd service username|
```
Comment (Amber1990Zhang) [inline src/tools/importer/README.md]: Ditto.

File: src/tools/importer/README.md [hunk 4]
```diff
@@ -1,33 +1,161 @@
...
+202,"Jane",17,"female"
+```
+
+Example command:
+
+```
+> java -jar importer-${VERSION}.jar --address 127.0.0.1:3699 --name myspace_test2 --schema student -u user -p password -t vertex --file data.txt --column name,age,gender --batch 2
+```
+
+E2. Insert `select` edges.
```
Comment (Amber1990Zhang) [inline src/tools/importer/README.md]: insert edges selected or `select` is an edge type?  I'm confusing. I think 
**Insert selected edges will be better.**

Comment (jude-zhu) [reply src/tools/importer/README.md]: In getStarted.md, there is an edge type named `select`

**Pull request merged**